### PR TITLE
Add recipe for S3Bundle

### PR DESCRIPTION
**Pull request by tcheymol**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: reconnect/s3bundle/0.1/config/packages/s3_bundle.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+s3_bundle:
```
Comment (Nyholm) [inline reconnect/s3bundle/0.1/config/packages/s3_bundle.yaml:1]: Could you please rename this file to reconnect_s3_bundle.yaml?

**Review by Nyholm — CHANGES_REQUESTED** [review]
Thank you. Just a minor

File: reconnect/s3bundle/0.1/config/packages/s3_bundle.yaml (hunk 1 above)
Comment (tcheymol) [reply reconnect/s3bundle/0.1/config/packages/s3_bundle.yaml]: @Nyholm It has been done, and the bundle updated, thanks

**Review by Nyholm — APPROVED** [review]
Thank you.

**Pull request merged**